src/js/components/githubUsers/button.tsx

```diff
@@ -33,6 +37,13 @@ const GitHubUserButton = ({
         <>
           <GitHubUserAvatar user={user} />
           <span className="collaborator-username">{name}</span>
+          {showPermissions && !user.permissions?.push && (
+            <Badge
+              content={i18n.t('read-only')}
+              color="light"
+              className="slds-m-left_xx-small"
```

Comment:
@jgerigmeyer How would you feel about increasing this margin to .5 rem by using slds-m-left_x-small instead of xx-small? 

I feel like it is a little tight in Chrome at least. Also, I played with the idea of having it right aligned, which is nice if there will be many folks who may be read only (easier to scan). Im not really tied to one way or the other, but I thought I would throw it out there. 

![metecho-right-aligned-badge](https://user-images.githubusercontent.com/1581694/115813920-b8d24f80-a3b9-11eb-9e56-2cd0129ab463.jpg)


Also, "read only" badge is supposed to only show up in the Add Collaborators modal and not after selected in the sidebar card, right? Wanted to confirm I should only see it in one place and not the other. 


![metecho-sidebar](https://user-images.githubusercontent.com/1581694/115813921-ba037c80-a3b9-11eb-9086-0880380d45f9.jpg)

Reply:
@stacyk 

1. I'm fine with slightly more padding; added.
2. I like the idea of right-aligning, if you want to go ahead and do that.
3. I do think it'd be a good idea to show "read-only" on the cards as well... but I'm not sure how to do that design-wise. There's often not enough space in the header (esp now that we've added full names). Does it look odd on its own line?

![Screen Shot 2021-04-23 at 11 10 27 AM](https://user-images.githubusercontent.com/552316/115891980-b6590f80-a424-11eb-8e84-88be8ab7eb73.png)

Reply:
@jgerigmeyer the other ideas I have:
1. Allow cards to take up the full row instead of 1/2 like at smaller size windows
2. Not a good idea, but we can truncate the name further, but then I would ask, what is more important, knowing that this person is a read-only collaborator or knowing the name of this person if you can't tell from the avatar. I think the avatar and name are the two most important (even more than username?). I don't know if username or read-only is more important though.

Reply:
@stacyk I went with both options, and structured the info as: `Full Name (username) <read-only>`

I think this is only missing the right-alignment of badges in the list (`<GitHubUserButton>`, used in `<UserTableCell>` in `assignEpicCollaborators.tsx`), if you still want to do that?

![Screen Shot 2021-04-26 at 11 25 46 AM](https://user-images.githubusercontent.com/552316/116109233-99ba1300-a682-11eb-9eda-36df5de74882.png)

![Screen Shot 2021-04-26 at 11 26 00 AM](https://user-images.githubusercontent.com/552316/116109236-99ba1300-a682-11eb-9536-416a0b8e90a4.png)